docs/languages/powershell.md

```diff
@@ -0,0 +1,132 @@
+
+PowerShell provides a powerful toolset for administrators on any platform.
+
+[The PowerShell Extension](https://github.com/PowerShell/vscode-powershell) provides rich language Support and IDE capabilities such as completions, definition tracking and linting analysis for Windows PowerShell versions 3, 4, 5 and 5.1 as well as all versions of PowerShell Core.
+
+## Install the PowerShell Extension on Visual Studio Code
+
+The official PowerShell Extension can be installed by following the steps in the
+[Visual Studio Code Docs](https://code.visualstudio.com/docs/editor/extension-gallery)
+or by going directly to the Marketplace and clicking the [Install Button](vscode:extension/ms-vscode.PowerShell)
+
+You can also install the PowerShell Extension from within Visual Studio Code by
+browsing the Extension Marketplace under the Marketplace view in the Activity Bar for extensions.
+You can access this with the shortcut <kbd>ctrl</kbd>-<kbd>Shift</kbd>-<kbd>x</kbd> or by clicking on the
+
+ Marketplace Icon:
+
+![Marketplace Activity Icon](./images/powershell/extensionBrowser.png)
+
+Search the Marketplace View for "PowerShell" and select the PowerShell Extension,
+Which is denoted by this Icon:
+
+![PowerShell Extension](./images/powershell/pwshVsixIcon.png)
+
+ ## Search from the Command line
+
+ Alternatively, the PowerShell Extension can be installed from any command line (including PowerShell, CMD, bash) on all platforms with the following command
+ 
+ ```code --install-extension ms-vscode.powershell```
+
+ If you are running VSCode Insiders, you will need this command instead:
+
+ ``` code-insiders --install-extension ms-vscode.powershell```
+
+## Example Scripts
+
+Example scripts are included with the extension and can be found at the following Path.
+
+    C:\Users\<yourusername>\.vscode\extensions\ms-vscode.PowerShell-<version>\examples
+
+To open or view the examples in Visual Studio Code run the following from your PowerShell command prompt
+
+```code (Get-ChildItem $Home\.vscode\extensions\ms-vscode.PowerShell-*\examples)[-1]```
+
+If using the Insiders edition
+
+```code-insiders (Get-ChildItem $Home\.vscode\extensions\ms-vscode.PowerShell-*\examples)[-1]```
+
+You can also open them with <kbd>ctrl</kbd>-<kbd>shift</kbd>-<kbd>p</kbd> (<kbd>cmd</kbd>-<kbd>shift</kbd>-<kbd>p</kbd>) for Macs and typing __PowerShell: Open Examples Folder__.
+
+![Open PowerShell Examples](./images/powershell/pwshExamples.png)
+
+## The PowerShell Extension includes the following major features
+
+* Syntax Highlighting
+* Code Snippets
+* Intellisense for cmdlets and more
+* Rule-Based analysis provided by PowerShell Script Analyzer
+* Go to definition of cmdlets and variables
+* Find references of cmdlets and variables
+* Document and Workspace symbol discovery
+* Run selected selection of PowerShell code using F8
+* Launch online help for the symbol under the cursor using ctrl+F1
+* Local script debugging and basic interactive console support
+
+## Plaster
```

Comment:
There's a Plaster integration in the VSCode extension that should probably be called out. If you open the command pallet and type "Plaster" you should see it. Just say that there's an integration, how to open it, and then tell the user to "follow the steps"